src/libs/actions/Report.js

```diff
@@ -1503,9 +1321,6 @@ Onyx.connect({
 });
 
 export {
-    fetchAllReports,
-    fetchOrCreateChatReport,
-    fetchChatReportsByIDs,
     fetchIOUReportByID,
```

Comment:
Possibly out of scope, but I wonder if `fetchIOUReportByID` & `fetchIOUReport` should also be migrated to a new API call. (something like `openPaymentDetailsPage`)

Reply:
Yep I agree. Definitely a refactor for another time though 😄